seed/views/v3/users.py

```diff
@@ -300,6 +301,8 @@ def create(self, request):
             role = body['role']
             if isinstance(role, dict):
                 role = role['value']
+            elif role == 'string':
```

Comment:
I could go one of two ways here:
1. Add a default for any value that is received here that we don't recognize. Maybe within the `_get_role_from_js` method?
2.  Improve this check to throw a 400 error that lists the valid role string values. Also, maybe within the `_get_role_from_js` method.

Reply:
I think option 2 just mimics the functionality of a required field because we would throw a 400 error every time a user doesn't explicitly define viewer, member, or owner.

Reply:
I was thinking it'd slightly be different in that it'd still be optional and they can exclude it from the request. But if an API user does provide a role, we do verify it's validity.